### DISCUSSION TITLE
skin.conf

### DISCUSSION THREAD

**Discussion by devonian** [Daveiano/weewx-wdc#153] [Q&A]
I installed as per the installtion instructions ( https://github.com/Daveiano/weewx-wdc#installation) and get the main index.html page but the links to Archive & NOAA Reports etc on the left frame don't work (links to root only and not to subfolder(s)).

I have weewx installed on a R-Pi + Raspbian with a 'vanilla' weewx install via setup.py - https://weewx.com/docs/setup.htm
In my weewx.conf, the html root is defined as HTML_ROOT = public_html (which translates to /home/weewx/public_html)
I use lighttpd on the R-Pi and have the lighttpd server document root set to /home/weewx/public_html
I tried editing the weewx-wdc skin.conf to the full path to my folders (/home/weewx/public_html) but that throws everything off.

Why not make the weewx-wdc skin.conf 'follow' where weewx.conf has defined the html_root - is that possible?

Otherwise, I like what I see, but as it stands, it no worky and I have reverted to the standard Seasons skin :(

Regards,
Nigel

**Comment by Daveiano** [comment]
Hi Nigel,

I can only assume what you did without seeing any of your conf files, but I assume you set `base_path` in skin.conf to something like `/home/weewx/public_html`, right?

There is no need for that and I wonder whats the intention behind this? The wiki states (https://github.com/Daveiano/weewx-wdc/wiki/Configuration#extras): 

> base_path Set the base path for your weewx-wdc installation. If the skin is deployed to the root of your webserver, you do not need to change this. If you are hosting the skin in eg: www.your-domain.com/weewx/wdc-skin/, the basePath should be set to /weewx/wdc-skin/.

You only need to change this if you are accessing your report files in a subfolder of your webserver like in the example above. 

Perhaps the description right above in the skin.conf could be more descriptive or could include an example to make it clear? The text currently is this:

```
# If weewx is installed in some sub-folder of your web server,
# please specify the path here.
base_path = /
```

**Comment by devonian** [comment]
Hi,
Correct on editing the skin.conf, base_path and adding /home/weewx/public_html

I figured it out and simply added /weewx/ in my skin.conf base_path and reverted my lighttpd to point to it's default as I had a 'weewx' simlink to the weewx default pages.
I also stopped weewx and removed all existing files in .../public_html  (sudo rm -rf /home/weewx/public_html) and rebuilt the files (wee_reports) and when that completed, started weewx again.
Pretty sure that removing and rebuilding the html files is what fixed it for me as I had a bunch of weewx default 'Seasons' files in there?!

Seems OK now ;-)

Need to figure out how the edit the files to add my VP2 ISS battery status (got some issues with it so like to keep and eye on it), maybe remove the heat index box and make it battery status or is there a nicer way to add a simple battey OK or LOW status?

Nigel

**Comment by Daveiano** [comment]
> I figured it out and simply added /weewx/ in my skin.conf base_path and reverted my lighttpd to point to it's default as I had a 'weewx' simlink to the weewx default pages.
I also stopped weewx and removed all existing files in .../public_html (sudo rm -rf /home/weewx/public_html) and rebuilt the files (wee_reports) and when that completed, started weewx again.
Pretty sure that removing and rebuilding the html files is what fixed it for me as I had a bunch of weewx default 'Seasons' files in there?!
> Seems OK now ;-)

Yeah, this could be the reason, but a wrong `base_path` could also lead to unexpected results. Happy it is working for you now!

> Need to figure out how the edit the files to add my VP2 ISS battery status (got some issues with it so like to keep and eye on it), maybe remove the heat index box and make it battery status or is there a nicer way to add a simple battey OK or LOW status?

The skin does not yet include telemetry data (observations like `consBatteryVoltage` or `rxCheckPercent`) - there is an ongoing discussion about it in #140 - feel free to add ideas/wishes.

**Comment by devonian** [comment]
I will close this as it is fixed, thanks.
Will join in on the other conversations as I move forward...

Nigel
